src/style.css

```diff
@@ -2,0 +1,99 @@
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: normal;
+  font-size: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: bold;
+  font-size: normal;
```

Comment:
Similar to the previous `@font-face` rule, the `font-size` property is not valid here and should be removed.